src/mdx-pages/guides/components.mdx

```diff
@@ -90,13 +96,12 @@ myButton.addClass('vjs-visible-text');
 
 ## Creating a Component
 
-Video.js components can be inherited and registered with Video.js to add new features and UI to the player.
+Video.js components can be extended and registered with Video.js to add new features and UI to the player.
 
-There are a few used for creating and registering components:
+There are a few functions used for creating and registering components:
 
-* `videojs.getComponent(String name)`: Retrieves component constructors from Video.js.
-* `videojs.registerComponent(String name, Function Comp)`: Registers component constructors with Video.js.
-* `videojs.extend(Function component, Object properties)`: Provides prototype inheritance. Can be used to extend a component's constructor, returning a new constructor with the given properties. *Deprecated, will be removed in Video.js 8 in favor of ES6 classes.
+* `videojs.getComponent(String name)`: Retrieves component classes from Video.js.
+* `videojs.registerComponent(String name, Function Comp)`: Registers component classes with Video.js.
 
 This example creates a title bar component, as a class extending `Component`.
```

Comment:
We should probably change the name of this component now that this is a builtin.

Reply:
Good point! 😆

Reply:
I think it's fine to leave as is, it serves an illustrative purpose regardless of potential naming collissions.